pbsmetrics/prometheus/prometheus.go

```diff
@@ -45,7 +45,6 @@ func NewMetrics(cfg config.PrometheusMetrics) *Metrics {
 	adapterLabelNames := []string{"demand_source", "request_type", "browser", "cookie", "adapter_bid", "adapter"}
 	bidLabelNames := []string{"demand_source", "request_type", "browser", "cookie", "adapter_bid", "adapter", "bidtype", "markup_type"}
 	errorLabelNames := []string{"demand_source", "request_type", "browser", "cookie", "adapter_error", "adapter"}
-
 	impLabelNames := []string{"banner_imps", "video_imps", "audio_imps", "native_imps"}
```

Comment:
Comparing to the latest master, I see all these label names have been converted to constants ... probably should do the same for `impLabelNames`

Reply:
Also can drop `_imps` from the label name, as these are labels on the `imps_requested_total` counter. Looking at `imps_requested_total.banner_imps` looks a bit redundant. :)